app/routes.js

```diff
@@ -0,0 +1,27 @@
+const { Router } = require('express')
+const Joi = require('@hapi/joi')
+const routes = Router()
+const UserController = require('./users/user.controller')
+const HttpCodes = require('./httpcodes')
+
+const userSchema = Joi.object({
+  userName: Joi.string()
+    .alphanum()
+    .required()
+})
+
+routes.post('/users', (request, response) => {
+  const { error } = userSchema.validate(request.body)
```

Comment:
Validation of requests is probably something that you'll want to do in a more generic way. For that you could define a validation middleware so you can send the schema to validate as a parameter and let it do its magic. You can start checking express documentation for middlewares: https://expressjs.com/en/guide/using-middleware.html